third_party/3/pyspark/sql/readwriter.pyi

```diff
@@ -11,14 +11,15 @@ from pyspark.sql.types import *
 
 PathOrPaths = Union[str, List[str]]
 TupleOrListOfString = Union[List[str], Tuple[str, ...]]
+FloatBoolOrString = Union[float, bool, str]
```

Comment:
Thanks for the PR @yilin-sai. 

In this case I'd go with inline annotation.

In general I use aliases in two cases:

- Complex unions with generic types (as these are painful to write, and easy to make a mistake).
- Unions of simple types, if these are reused across multiple modules (these usually go to respective virtual `_typing` module).